operator/controllers/config/config.go

```diff
@@ -23,17 +24,20 @@ func NewInterceptorFromEnv() (*Interceptor, error) {
 	}
 	adminPort := env.GetInt32Or("KEDAHTTP_OPERATOR_INTERCEPTOR_ADMIN_PORT", 8090)
 	proxyPort := env.GetInt32Or("KEDAHTTP_OPERATOR_INTERCEPTOR_PROXY_PORT", 8091)
+	pullPolicy := env.GetOr("INTERCEPTOR_PULL_POLICY", "Always")
```

Comment:
@khaosdoctor what are your thoughts on checking this for a valid value (i.e. `Always`, etc...)?